cnxepub/tests/test_collation.py

```diff
@@ -280,4 +280,4 @@ def test_with_ruleset(self):
         # Check for the appended composite document
         self.assertEqual(len(collated_binder), 3)
         self.assertEqual(collated_binder[2].metadata['title'],
-                         'Composite One')
+                         'Document One')
```

Comment:
The problem appears to be, if we don't ignore the TOC, easybake is not going to work.  I recommend adding `FIXME` and commenting out the code instead of removing it.